.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,11 +10,31 @@ labels: kind/bug
   Before reporting a bug, make sure PHP CS Fixer is up-to-date.
 
   Check that the issue is NOT regarded to an applied rule by using the `-vvv` option (will list applied rules) on a minimal code snippet.
+-->
+
+### Runtime version
```

Comment:
Did you mean something like _Description_?

Reply:
🤦🏻 thanks

```suggestion
### Description
```